### PR TITLE
feat: remove old UserPage, related older actions and version toggle

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**
[PROD-2761](https://openedx.atlassian.net/browse/PROD-2761)
Removes old User Page, all related older actions and toggle button which was was used as a rollback option in the MFE.
Once this merges, I'll refactor all the v2 components back as default.

### CODE REVIEW

File: src/users/v2/UserPage.jsx [hunk 1]
```diff
@@ -1,171 +0,0 @@
...
-} from 'react';
-import PageLoading from '../../components/common/PageLoading';
-import AlertList from '../../userMessages/AlertList';
-import { USER_IDENTIFIER_INVALID_ERROR } from '../../userMessages/messages';
-import UserMessagesContext from '../../userMessages/UserMessagesContext';
-import { isEmail, isValidUsername, isValidLMSUserID } from '../../utils/index';
-import { getAllUserData } from '../data/api';
-import UserSearch from '../UserSearch';
-import LearnerInformation from './LearnerInformation';
-import { LEARNER_INFO_TAB, TAB_PATH_MAP } from '../../SupportToolsTab/constants';
```
Comment (DawoudSheraz) [inline src/users/v2/UserPage.jsx]: Why we are deleting v2 userpage?

Comment (Ali-D-Akbar) [reply src/users/v2/UserPage.jsx:14]: I moved v2/UserPage to actual directory.

Comment (Ali-D-Akbar) [reply src/users/v2/UserPage.jsx]: I moved v2/UserPage to parent directory.

File: src/users/data/test/onboardingStatus.js [hunk 2]
```diff
@@ -1,12 +1,3 @@
-const onboardingStatusData = {
-  onboardingStatus: 'verified',
-  expirationDate: null,
-  onboardingLink: '/course/course-uuid/some-route',
-  onboardingPastDue: false,
-  onboardingReleaseDate: new Date().toISOString(),
-  reviewRequirementsUrl: null,
-};
-
 const v2OnboardingStatusData = {
```
Comment (ansabgillani) [inline src/users/data/test/onboardingStatus.js:1]: Rename this as onboardingStatusData and change the statuses

File: src/users/v2/OnboardingStatus.test.jsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { waitForComponentToPaint } from '../../setupTest';
 import OnboardingStatus from './OnboardingStatus';
 import UserMessagesProvider from '../../userMessages/UserMessagesProvider';
-import { v2OnboardingStatusData as onboardingStatusData } from '../data/test/onboardingStatus';
+import v2OnboardingStatusData from '../data/test/onboardingStatus';
```
Comment (ansabgillani) [inline src/users/v2/OnboardingStatus.test.jsx:6]: Why did we remove this alias?

File: src/users/v2/OnboardingStatus.test.jsx [hunk 4]
```diff
@@ -109,7 +113,7 @@ describe('Onboarding Status', () => {
       enrollmentDate: null,
       instructorDashboardLink: null,
     };
-    const onboardingData = { ...onboardingStatusData, verifiedIn: nullData };
+    const onboardingData = { ...v2OnboardingStatusData, verifiedIn: nullData };
```
Comment (ansabgillani) [inline src/users/v2/OnboardingStatus.test.jsx:116]: Can you update this v2OnboardingStatusData to onboardingStatusData? There is no sense in keeping it named as v2 if we're deprecating.

**Review by ansabgillani — CHANGES_REQUESTED** [review]
Left a few comments. please look onto them.

File: src/users/data/test/onboardingStatus.js (hunk 2 above)
Comment (Ali-D-Akbar) [reply src/users/data/test/onboardingStatus.js:1]: will be done in the next PR.

File: src/users/v2/OnboardingStatus.test.jsx (hunk 3 above)
Comment (Ali-D-Akbar) [reply src/users/v2/OnboardingStatus.test.jsx:6]: bcz there is no need of it anymore. The `onboardingStatusData` has been removed from `../data/test/onboardingStatus`

File: src/users/v2/OnboardingStatus.test.jsx (hunk 4 above)
Comment (Ali-D-Akbar) [reply src/users/v2/OnboardingStatus.test.jsx:116]: will be done in the next PR.

**Review by ansabgillani — APPROVED** [review]

**Review by azanbinzahid — APPROVED** [review]

**Review by DawoudSheraz — APPROVED** [review]

**Pull request merged**